v03_pipeline/lib/tasks/reference_data/update_variant_annotations_table_with_updated_reference_dataset.py

```diff
@@ -0,0 +1,59 @@
+import hail as hl
+import luigi
+
+from v03_pipeline.lib.model import ReferenceDatasetCollection
+from v03_pipeline.lib.reference_data.compare_globals import (
+    Globals,
+    GlobalsValidator,
+)
+from v03_pipeline.lib.tasks.base.base_variant_annotations_table import (
+    BaseVariantAnnotationsTableTask,
+)
+
+
+class UpdateVariantAnnotationsTableWithUpdatedReferenceDataset(
+    BaseVariantAnnotationsTableTask,
+):
+    rdc = luigi.EnumParameter(enum=ReferenceDatasetCollection)
+
+    def complete(self) -> bool:
```

Comment:
Is this method gonna work correctly if the reference dataset collection has been correctly updated and persisted before this task is run?

Reply:
not sure what you mean by this, got a sec before 5 to chat?

Reply:
I went through and re-read this before our chat so I could re-fresh myself... I think this is actually correct as is and I misunderstood it before!  sorry for the confusing question!